packages/angular/projects/clr-angular/src/accordion/stepper/stepper-panel.ts

```diff
@@ -56,7 +56,7 @@ export class ClrStepperPanel extends ClrAccordionPanel implements OnInit {
   }
 
   constructor(
-    @Inject(PLATFORM_ID) private platformId: Record<string, any>,
+    @Inject(PLATFORM_ID) private platformId: any,
```

Comment:
Was the Record typing causing an issue?

Reply:
The stricter Angular compiler setting was unable to have a generic in the typing for something explicitly using `@Inject()`. There is a bug for it, but would have to dig it up.